README.md

```diff
@@ -48,33 +48,41 @@ Install each of these and ensure they are included in your PATH. Then install th
-| --metadata_json            | Json file that describe the project, analysis, samples and files                                                                                                                                                   |
-| --vcf_files                | One or several vcf files to validate.This allows you to provide multiple VCF files to validate and a single associated reference genome file. The VCF files and the associated reference genome file must use the same chromosome naming convention                                                                                                                                                                              |
-| --reference_fasta          | The fasta file containing the reference genome from which the variants were derived                                                                                                                                |
-| --submission_dir           | Path to the directory where all processing will be done and submission data is/will be stored                                                                                                                      |
-| --tasks {validate,submit}  | Selecting VALIDATE will run the validation regardless of the outcome of previous runs. Selecting SUBMIT will run validate only if the validation was not performed successfully before and then run the submission |
-| --executor {docker,native} | Select an execution type for running validation (default native)                                                                                                                                                   |
-| --shallow                  | Set the validation to be performed on the first 10000 records of the VCF. Only applies if the number of record exceed 10000                                                                                        |
-| --username                 | Username used for connecting to the ENA webin account                                                                                                                                                              |
-| --password                 | Password used for connecting to the ENA webin account                                                                                                                                                              |
+### Submission directory
 
+This is the directory where all processing will take place, and where configuration and reports will be saved.
+Crucially, the eva-sub-cli tool requires that there be **only one submission per directory**.
+Running multiple submissions from a single directory can result in data loss during validation and submission.
```

Comment:
@tcezard Based on the discussion this morning, do you think this is clear enough? I was going to add something about not reusing a directory, but I think that's the same thing as running multiple from a single directory..

Reply:
updated to clarify reusing is also not allowed

Reply:
Makes sense